src/components/gataca-qrws/gataca-qrws.scss

```diff
@@ -99,175 +105,202 @@ img {
     transition: opacity 0.5s, visibility 0s 0.5s;
     visibility: hidden;
     opacity: 0;
+
     &__content {
       width: 100%;
       height: 100%;
       text-align: left;
+
+      .hidenText {
+        display: none;
+      }
     }
-    p {
-      font-family: "Work Sans", "Helvetica Neue", sans-serif;
-      font-weight: normal;
-      font-size: 12px;
+
+    .modalText {
+      font-family: "Ubuntu", "Work Sans", "Helvetica Neue", sans-serif;
+      font-size: 20px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 24px;
+      margin-top: 10px;
       letter-spacing: 1px;
-      color: #1e1e20;
     }
-    .qrTitleContainer{
+
+    .qrTitleContainer {
       padding: 14px 24px;
       padding-bottom: 0px;
-      //margin-bottom: 20px;
     }
-    .qrTitle{
+
+    .qrTitle {
       font-size: 20px;
       font-weight: 600;
       line-height: 23.5px;
     }
-    .qrDescription{
-      font-weight: 400;
-      font-size: 14px;
-      line-height: 16.5px;
-      padding-top: 12px;
-      padding-bottom: 5px;
 
+    .qrDescription {
+      font-family: Poppins;
```

Comment:
This font calling isn't correct, should be "Poppins", also change for gacata-qr .. 

![Captura de pantalla 2024-03-22 a las 19 53 21](https://github.com/gataca-io/gataca-QR/assets/127114451/67eedb13-ffe4-403d-bc58-771623453c00)

Reply:
You're right. I don't know why it was loading in my test. I've also put others fonts of backup.